### PR TITLE
Feature/eco 161 campaign manager

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/SprykerEco/Yves/FactFinder/DataProvider/ShoppingCartCampaignsDataProvider.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+use Generated\Shared\Transfer\FactFinderSdkProductCampaignRequestTransfer;
+
+class ShoppingCartCampaignsDataProvider extends ProductCampaignsDataProvider
+{
+
+    /**
+     * @param array $parameters
+     *
+     * @return array|\Generated\Shared\Transfer\FactFinderSdkDataCampaignIteratorTransfer
```
Comment (dereuromark) [inline src/SprykerEco/Yves/FactFinder/DataProvider/ShoppingCartCampaignsDataProvider.php:18]: Sure it can be both?

Comment (ghost) [reply src/SprykerEco/Yves/FactFinder/DataProvider/ShoppingCartCampaignsDataProvider.php:18]: I'm sure it can't) I'll fix in a min

Comment (dereuromark) [reply src/SprykerEco/Yves/FactFinder/DataProvider/ShoppingCartCampaignsDataProvider.php:18]: you can always assign it to someone for (quick) review.
4 eyes see more then 2 :)

Comment (dereuromark) [reply src/SprykerEco/Yves/FactFinder/DataProvider/ShoppingCartCampaignsDataProvider.php:18]: You can also directly remove merged branches after merge.

Comment (ghost) [reply src/SprykerEco/Yves/FactFinder/DataProvider/ShoppingCartCampaignsDataProvider.php:18]: Yeah, I'll remove all of them soon :)

**Pull request merged**